OpenMage/magento-lts
# Cleanup: disable currencyconverterapi in config

### Description (*)
Update config file to disable currencyconverterapi service by default

### Related Pull Requests
----

### Fixed Issues (if relevant)

1. Fixes OpenMage/magento-lts#1093

### Manual testing scenarios (*)

1. Login to Backend -> System -> Configuration -> Currency setup
2. Open CurrencyConverterAPI Tab
3. Enabled should be "no"


### Contribution checklist (*)
